package.json

```diff
@@ -24,13 +29,16 @@
     "tmconvert": "node scripts/tmconvert.js",
     "version": "npm run compile && npm run test",
     "postversion": "git push && git push --tags",
-    "prepublishOnly": "tsc && webpack --progress",
-    "bundle": "webpack"
+    "prepublishOnly": "tsc && webpack --progress && npm run api-extractor",
+    "bundle": "webpack",
```

Comment:
Should `npm run api-extractor` be also invoked by the `bundle` script? The build files seem to invoke `npm run bundle` and I don't know if `prepublishOnly` is invoked as part of our npm package publishing. Maybe we can trigger a build to check the package contents? But that would require that you port your changes to a branch in this repository, since forks can't be built.